spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/LogbackLoggingSystem.java

```diff
@@ -196,6 +206,28 @@ private void addLevelChangePropagator(LoggerContext loggerContext) {
 		loggerContext.addListener(levelChangePropagator);
 	}
 
+	private void registerJmxConfigurator(LoggerContext loggerContext,
+			LoggingInitializationContext initializationContext, String configLocation,
+			LogFile logFile) {
+		String objectNameAsStr = MBeanUtil.getObjectNameFor(
```

Comment:
This is duplicating code from LogBack which enables the MBean with a single line in its XML configuration. Shouldn't we try to reduce the amount of code on our side for this?